### ISSUE TITLE
Windows builds no longer pass "-H windowsgui"

### ISSUE THREAD

**Issue by Jacalz** [fyne-io/fyne-cross#97]
### Describe the bug:

Windows binaries seems to no longer pass "-H windowsgui", which means that all binaries (even those built without `-console`) show a console when opened. This was recently reported to me at https://github.com/Jacalz/rymdport/issues/34.

I suspect that this broke with https://github.com/fyne-io/fyne-cross/pull/42/files#r692376503 but I sadly don't have time to look at it right now.

### To Reproduce:
Steps to reproduce the behaviour:
1. Build a binary for windows without the `-console` flag.
2. Run the binary on Windows and notice that console is shown.

### Example code:
Any code will do.

### Device and debug info (please complete the following information):

<summary>Device info</summary>

 - **OS:**  Fedora Workstation
 - **Version:**  35 with kernel 5.16.17
 - **Go version:**  1.16.15
 - **fyne-cross version:**  1.2.0
 - **Fyne version:**  2.1.3 (not relevant)

<details><summary>Debug info</summary><br><pre>
I did not bother with --no-cache because `--debug` has enough meaningful information.
```bash
[jacob@vostro rymdport]$ ~/go/bin/fyne-cross windows -arch amd64 --debug
[i] Target: windows/amd64
command.Context{Volume:volume.Volume{binDirHost:"/home/jacob/Git/rymdport/fyne-cross/bin", cacheDirHost:"/home/jacob/.cache/fyne-cross", distDirHost:"/home/jacob/Git/rymdport/fyne-cross/dist", tmpDirHost:"/home/jacob/Git/rymdport/fyne-cross/tmp", workDirHost:"/home/jacob/Git/rymdport"}, Architecture:"amd64", Env:[]string{"GOOS=windows", "GOARCH=amd64", "CC=x86_64-w64-mingw32-gcc"}, ID:"windows-amd64", LdFlags:[]string{"-H=windowsgui"}, OS:"windows", Tags:[]string(nil), AppBuild:"16", AppID:"io.github.jacalz.rymdport", AppVersion:"3.0.0", CacheEnabled:true, DockerImage:"docker.io/fyneio/fyne-cross:1.2-windows", Icon:"internal/assets/icon/icon-512.png", Name:"rymdport.exe", Package:".", Release:false, StripDebug:true, Debug:true, Pull:false, Category:"", Certificate:"", Developer:"", Keystore:"", KeystorePass:"", KeyPass:"", Password:"", Profile:""}
[i] Cleaning target directories...
[✓] "bin" dir cleaned: /home/jacob/Git/rymdport/fyne-cross/bin/windows-amd64
[✓] "dist" dir cleaned: /home/jacob/Git/rymdport/fyne-cross/dist/windows-amd64
[✓] "temp" dir cleaned: /home/jacob/Git/rymdport/fyne-cross/tmp/windows-amd64
[i] Checking for go.mod: /home/jacob/Git/rymdport/go.mod
[✓] go.mod found
/usr/bin/podman run --rm -t -w /app/fyne-cross/tmp/windows-amd64 -v /home/jacob/Git/rymdport:/app:z --userns keep-id -e use_podman=1 -e CGO_ENABLED=1 -e GOCACHE=/go/go-build docker.io/fyneio/fyne-cross:1.2-windows /usr/local/bin/gowindres -arch amd64 -output rymdport.exe -workdir /app/fyne-cross/tmp/windows-amd64
[i] Building binary...
[go build -trimpath -o /app/fyne-cross/bin/windows-amd64/rymdport.exe -v .]
/usr/bin/podman run --rm -t -w /app -v /home/jacob/Git/rymdport:/app:z -v /home/jacob/.cache/fyne-cross:/go:z --userns keep-id -e use_podman=1 -e CGO_ENABLED=1 -e GOCACHE=/go/go-build -e GOOS=windows -e GOARCH=amd64 -e CC=x86_64-w64-mingw32-gcc -e CGO_LDFLAGS=-w -s docker.io/fyneio/fyne-cross:1.2-windows go build -trimpath -o /app/fyne-cross/bin/windows-amd64/rymdport.exe -v .
[✓] Binary: /home/jacob/Git/rymdport/fyne-cross/bin/windows-amd64/rymdport.exe
[i] Packaging app...
[✓] Package: /home/jacob/Git/rymdport/fyne-cross/dist/windows-amd64/rymdport.exe.zip
```
</pre></details>

**Comment by Jacalz** [comment]
I suspect that this may be partly related to https://github.com/fyne-io/fyne-cross/issues/96.

**Comment by lucor** [comment]
Fix available for testing in #99

**Comment by Jacalz** [comment]
Will be fixed in the next release.
